### PR TITLE
issue-89 fixed status tag alignment on mobile

### LINKED ISSUE #89: HMRC Status tags don't left-align on mobile
The HMRC status tags are meant to behave in the same way as the Gov Frontend versions, but on smaller screens they don't stack left-aligned under the link.

Screenshot 2020-08-03 at 12 55 24
Screenshot 2020-08-03 at 12 54 33

To fix:
The `.hmrc-status-tag` class needs the `float: right` only applying for larger sizes

So instead of this
```
.hmrc-status-tag {
  float: right;
  color: $govuk-text-colour;
  font-family: "GDS Transport", Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
}
```

Have this
```
.hmrc-status-tag {
  color: $govuk-text-colour;
  font-family: "GDS Transport", Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
}
@media (min-width: 28.125em) {
    .hmrc-status-tag {
        float: right;
    }
}
```

### PR DESCRIPTION
**Pull request by timsb**
This fixes issue https://github.com/hmrc/hmrc-frontend/issues/89 on mobile width devices where status tags aren't left aligned. Some changes were made based on gov.uk task list styles at https://github.com/alphagov/govuk-prototype-kit/blob/master/app/assets/sass/patterns/_task-list.scss

### CODE REVIEW

File: src/components/status-tags-in-task-list-pages/_status-tags-in-task-list-pages.scss [hunk 1]
```diff
@@ -16,8 +24,13 @@
 }
 
 .app-task-list__task-name {
+  display: block;
   width: auto;
 
+  @include govuk-media-query($from: 450px) {
+    float: left;
+  }
+
```
Comment (adamliptrot-oc) [inline src/components/status-tags-in-task-list-pages/_status-tags-in-task-list-pages.scss]: As this is a change to the HMRC component, this shouldn't require an update to the Gov Frontend / GDS task list pattern css.

**Review by adamliptrot-oc — CHANGES_REQUESTED** [review]

File: src/components/status-tags-in-task-list-pages/_status-tags-in-task-list-pages.scss (hunk 1 above)
Comment (timsb) [reply src/components/status-tags-in-task-list-pages/_status-tags-in-task-list-pages.scss]: I've updated commit. Please review

**Review by adamliptrot-oc — APPROVED** [review]
Looks good

**Review by adamliptrot-oc — DISMISSED** [review]

**Review by adamliptrot-oc — APPROVED** [review]

**Review by mikeash82 — COMMENTED** [review]
seems fine.

**Review by mikeash82 — APPROVED** [review]

**Review by matthewmascord — APPROVED** [review]

**Pull request merged**